src/components/Navbar.jsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+
+const Navbar = () => {
+  return (
+    <header className="bg-white shadow-lg top-0 sticky">
+      <div className="container mx-auto flex flex-wrap p-5 justify-between items-center">
+        <a
+          href="/"
+          className="flex title-font font-medium items-center text-gray-900 mb-4 md:mb-0"
+        >
+          <svg
+            xmlns="http://www.w3.org/2000/svg"
+            fill="none"
+            stroke="currentColor"
+            strokeLinecap="round"
+            strokeLinejoin="round"
+            strokeWidth="2"
+            className="w-10 h-10 p-2 bg-indigo-500 text-white rounded-full"
+            viewBox="0 0 24 24"
+          >
+            <path d="M12 2L2 7l10 5 10-5-10-5zM2 17l10 5 10-5M2 12l10 5 10-5"></path>
+          </svg>
+          <span className="ml-3 text-2xl font-bold">WebRTC-Client</span>
```

Comment:
Remove underline from WebRTC-Client, Home, About Us

Replace WebRTC-Client to ZoomXF